oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/record-requests/components/CreateRequestButtonGroup.jsx

```diff
@@ -1,63 +1,139 @@
+  const createRequests = requestTypes?.filter(
+    (requestType) => requestType.links.actions?.create
+  );
+  const isMutating = useIsMutating();
 
-/**
- * @param {{ requestTypes: RequestType[], isLoading: boolean, loadingError: Error }} props
- */
-export const CreateRequestButtonGroup = ({ recordLoading, recordLoadingError }) => {
-  const { requestTypes } = useRequestContext();
-  const createRequests = requestTypes.filter(requestType => requestType.links.actions?.create);
+  let content;
 
-  return (
-    <Segment className="requests-create-request-buttons borderless">
-      <Header size="small" className="detail-sidebar-header">{i18next.t("Requests")}</Header>
-      {recordLoading ?
-        <Placeholder>
-          {Array.from({ length: 2 }).map((_, index) => (
-            <Placeholder.Paragraph key={index}>
-              <Icon name="plus" disabled />
-            </Placeholder.Paragraph>
-          ))}
-        </Placeholder> :
-        recordLoadingError ?
-          <Message negative>
-            <Message.Header>{i18next.t("Error loading request types")}</Message.Header>
-            <p>{recordLoadingError?.message}</p>
-          </Message> :
-          !_isEmpty(createRequests) ?
-            <Button.Group vertical compact fluid>
-              {createRequests.map((requestType) => {
-                const header = !_isEmpty(requestType?.title) ? requestType.title : (!_isEmpty(requestType?.name) ? requestType.name : requestType.type);
-                const modalActions = mapLinksToActions(requestType);
-                return (
-                  <RequestModal
-                    key={requestType.type_id}
-                    requestType={requestType}
-                    header={header}
-                    trigger={
-                      <Button icon="plus" className="pl-0" title={i18next.t(requestType.name)} basic compact content={requestType.name} />
-                    }
-                    actions={modalActions}
-                    ContentComponent={CreateRequestModalContent}
-                  />
-                )
-              }
-              )}
-            </Button.Group> :
-            <p>{i18next.t("No new requests to create")}.</p>
+  if (applicableRequestsLoading) {
+    content = (
+      <Placeholder>
+        {Array.from({ length: 2 }).map((_, index) => (
+          <Placeholder.Paragraph key={index}>
+            <Placeholder.Line length="full" />
+            <Placeholder.Line length="medium" />
+          </Placeholder.Paragraph>
+        ))}
+      </Placeholder>
+    );
+  } else if (applicableRequestsLoadingError) {
+    content = (
+      <Message negative className="rel-mb-1">
+        <Message.Header>
+          {i18next.t("Error loading request types")}
+        </Message.Header>
+      </Message>
+    );
+  } else if (_isEmpty(createRequests)) {
+    return null; // No need to render anything if there are no requests
+  } else {
+    content = createRequests.map((requestType) => {
```

Comment:
This `requestType` item render wants to be extracted to separate `RequestButton` component :) to keep CRBGroup comp shorter